pkgs/by-name/go/gotenberg/package.nix

```diff
@@ -0,0 +1,100 @@
+{
+  lib,
+  buildGoModule,
+  chromium,
+  fetchFromGitHub,
+  libreoffice,
+  makeWrapper,
+  pdftk,
+  qpdf,
+  unoconv,
+  mktemp,
+  makeFontsConf,
+  liberation_ttf_v2,
+  jre17_minimal,
+  exiftool,
+  python311,
+  nix-update-script,
+}:
+let
+  fontsConf = makeFontsConf { fontDirectories = [ liberation_ttf_v2 ]; };
+  jre' = jre17_minimal.override {
+    modules = [
+      "java.base"
+      "java.desktop"
+      "java.logging"
+      "java.sql"
+    ];
+  };
+  libreoffice' = "${libreoffice}/lib/libreoffice/program/soffice.bin";
+  unoconv' = unoconv.override { python3 = python311; };
+in
+buildGoModule rec {
+  pname = "gotenberg";
+  version = "8.8.0";
+
+  src = fetchFromGitHub {
+    owner = "gotenberg";
+    repo = "gotenberg";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-OHvtdUFZYm+I8DN8lAdlNAIRLWNRC/GbVEueRfyrwDA=";
+  };
+
+  vendorHash = "sha256-w9Q3hK8d5NwDYp8kydrWrlBlOmMyDqDixdv3z79yhK8=";
+
+  patchPhase = ''
+    find ./pkg -name '*_test.go' -exec sed -i -e 's#/tests#${src}#g' {} \;
+  '';
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/gotenberg/gotenberg/v8/cmd.Version=${version}"
+  ];
+
+  checkInputs = [
+    chromium
+    libreoffice
+    pdftk
+    qpdf
+    unoconv'
+    mktemp
+    jre'
+  ];
+
+  preCheck = ''
+    export CHROMIUM_BIN_PATH=${lib.getExe chromium}
+    export PDFTK_BIN_PATH=${lib.getExe pdftk}
+    export QPDF_BIN_PATH=${lib.getExe qpdf}
+    export UNOCONVERTER_BIN_PATH=${lib.getExe unoconv'}
+    export EXIFTOOL_BIN_PATH=${lib.getExe exiftool}
+    # LibreOffice needs all of these set to work properly
+    export LIBREOFFICE_BIN_PATH=${libreoffice'}
+    export FONTCONFIG_FILE=${fontsConf}
+    export HOME=$(mktemp -d)
+    export JAVA_HOME=${jre'}
+  '';
+
+  checkFlags = [ "-skip=^TestChromiumBrowser_(screenshot|pdf)$" ];
+
+  preFixup = ''
+    wrapProgram $out/bin/gotenberg \
+      --set PDFTK_BIN_PATH "${lib.getExe pdftk}" \
+      --set QPDF_BIN_PATH "${lib.getExe qpdf}" \
+      --set UNOCONVERTER_BIN_PATH "${lib.getExe unoconv'}" \
+      --set EXIFTOOL_BIN_PATH "${lib.getExe exiftool}" \
+      --set JAVA_HOME "${jre'}"
+  '';
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "Converts numerous document formats into PDF files";
+    homepage = "https://gotenberg.dev";
+    repository = "https://github.com/gotenberg/gotenberg";
```

Comment:
```suggestion
```
this is not a standard attribute -- unless i'm missing something in the [docs](https://github.com/NixOS/nixpkgs/blob/a6c83f4952ad4e43176cb3bda4f2cf7e08ffd321/doc/stdenv/meta.chapter.md)

Reply:
It's a *proposed* attribute but it's been [bikeshedded to hell and back](https://github.com/NixOS/nixpkgs/issues/293838)

Reply:
Could have sworn it was accepted, I'll remove since it's still in the traditional bikeshedding phase.

Reply:
Removed it